composer.json

```
@@ -31,7 +31,8 @@
         "symfony/http-foundation": "^4.4 || ^5.0",
         "symfony/http-kernel": "^4.4 || ^5.0",
         "symfony/options-resolver": "^4.4 || ^5.0",
-        "twig/twig": "^2.9 || ^3.0"
+        "twig/twig": "^2.9 || ^3.0", 
+        "sonata-project/block-bundle": ">=3.0 <4.0"
```

```suggestion
        "sonata-project/block-bundle": "^3.11"
```
And you can remove it from the conflict section then.

And also, the dependencies have to be alphabetically ordered